Connectors/Rhino/Speckle.Connectors.RhinoShared/Bindings/RhinoSendBinding.cs

```diff
@@ -145,7 +145,12 @@ private void SubscribeToRhinoEvents()
 
   public List<ISendFilter> GetSendFilters() => _sendFilters;
 
-  public List<CardSetting> GetSendSettings() => [];
+#pragma warning disable CA1024
```

Comment:
@JR-Morgan do we still need this pragma and `return new()`? Isn't it `[]` is same?

Reply:
[CA1024: Use properties where appropriate](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1024) will not complain so long as your implementing an interface.

So it can be removed here, unrelated to the list stuff.

But fyi, `[]` will be have the same as` new List<T>()` for list typed variables/returns.